### DISCUSSION TITLE
Using cppflow with TFSecured

### DISCUSSION THREAD

**Discussion by JordanDavis-1** [serizba/cppflow#214] [General]
The current project I'm working on needs the ability for me to encrypt frozen .pb files and decrypt back within the app to run for inference . I came across this small library called TFSecured which allows exactly that: https://github.com/dneprDroid/TFSecured

Does anyone know how to go about integrating the TFSecured library to work with Cppflow?

**Comment by serizba** [comment]
Moving this to a discussion because there is no plan into integrating this. If anyone can help, it can post the answer in the discussion.
